data/src/main/java/com/example/android/data/repositories/ReviewRepository.kt

```diff
@@ -2,15 +2,15 @@ package com.example.android.data.repositories
 
 import com.example.android.data.sources.RemoteReviewDataSource
 import com.example.android.domain.Review
-import com.example.android.framework.data.remote.requests.Result
+import com.example.android.domain.result.DataResult
 
 class ReviewRepository(
     private val remoteReviewDataSource: RemoteReviewDataSource
 ){
 
     //region Public Methods
```

Comment:
Good implementation!

I didn't know that we can use regions in android studio.